js/languagePickerDrawerView.js

```diff
@@ -46,56 +46,45 @@ define([
         _showIcon: true
       };
 
-      //keep active element incase the user cancels - usually navigation bar icon
-      //move drawer close focus to #focuser
-      if ($.a11y) {
-        // old a11y API (Framework v4.3.0 and earlier)
-        this.$finishFocus = $.a11y.state.focusStack.pop();
-        $.a11y.state.focusStack.push($('#focuser'));
-      } else {
-        this.$finishFocus = Adapt.a11y._popup._focusStack.pop();
-        Adapt.a11y._popup._focusStack.push($('#a11y-focuser'));
-      }
-
-      Adapt.once('drawer:closed', function() {
-        //wait for drawer to fully close
-        _.delay(function(){
-          //show yes/no popup
-          Adapt.once('popup:opened', function() {
-            //move popup close focus to #focuser
-            if ($.a11y) {
-              // old a11y API (Framework v4.3.0 and earlier)
-              $.a11y.state.focusStack.pop();
-              $.a11y.state.focusStack.push($('#focuser'));
-              return;
-            }
+      // keep active element incase the user cancels - usually navigation bar icon
+      // move drawer close focus to #focuser
+      this.$finishFocus = Adapt.a11y._popup._focusStack.pop();
+      Adapt.a11y._popup._focusStack.push($('#a11y-focuser'));
+
+      Adapt.once('drawer:closed', () => {
+        // wait for drawer to fully close
+        _.delay(() => {
+          Adapt.once('popup:opened', () => {
+            // move popup close focus to #focuser
             Adapt.a11y._popup._focusStack.pop();
             Adapt.a11y._popup._focusStack.push($('#a11y-focuser'));
           });
-
-          Adapt.trigger('notify:prompt', promptObject);
+          // show yes/no popup
+          Adapt.notify.prompt(promptObject);
         }, 250);
       });
 
       Adapt.trigger('drawer:closeDrawer');
     },
 
     onDoChangeLanguage: function () {
-      // set default language
       var newLanguage = this.model.get('newLanguage');
       this.model.setTrackedData();
       this.model.setLanguage(newLanguage);
       this.remove();
     },
 
+    /**
+     * If the learner selects 'no' in the 'confirm language change' prompt,
+     * wait for notify to close completely then send focus to the
+     * navigation bar icon
+     */
     onDontChangeLanguage: function () {
       this.remove();
 
-      // wait for notify to close fully
-      _.delay(function(){
-        // focus on navigation bar icon
-        this.$finishFocus.a11y_focus();
-      }.bind(this), 500);
+      _.delay(() => {
+        Adapt.a11y.focusFirst(this.$finishFocus);
+      }, 500);
```

Comment:
One-liner?
```suggestion
      _.delay(() => Adapt.a11y.focusFirst(this.$finishFocus), 500);
```

Reply:
possibly... we could do with setting a style guide for these. I know eslint complains if you try to do something like this:
```js
_.defer(() => this._disableAnimationOnce = false);
```
Some people seem to favour only using the one line style when you're returning the value of an expression.

TBH I only left it like that cause it requires minimal effort to change to that style ;-)

Reply:
😆 fair enough!!